components/common/navbar/navbar.tsx

```diff
@@ -1,15 +1,38 @@
 
 import React, { FC } from 'react';
 import Link from 'next/link';
+import { useRouter } from 'next/router';
+
+const links = [
+    {
+        href: '/',
+        title: 'Home',
+    },
+    {
+        href: 'https://github.com/jarekb84/my',
+        title: 'GitHub',
+        target: '_blank',
+        rel: 'noreferrer',
+    },
+    {
+        href: '/resume',
+        title: 'Resume',
+    },
+];
 
 const Navbar: FC = () => {
-    return <nav className="navbar">
-        <Link href="/">
-            <a>Home</a>
-        </Link>
-        <Link href="/resume">
-            <a>Resume</a>
-        </Link>
+    const { asPath } = useRouter();
+
+    return <nav className="navbar status">
+        {links.map(({ href, title, target, rel }) => {
+            return <Link href={href} key={href}>
+                <a className={`navbar__link ${asPath === href ? 'navbar__link--active' : ''}`}
```

Comment:
If this get's more complex, I'll consider breaking this out into an `<ActiveLink />` component and focus this navbar component on just the list of links to show.